src/FiberTraces.cc

```diff
@@ -1047,6 +1047,8 @@ namespace pfsDRPStella = pfs::drp::stella;
       throw LSST_EXCEPT(pexExcept::Exception, message.c_str());
     }
 
+    double tmpVal;
```

Comment:
You shouldn't declare a variable until you're going to use it, and you should initialise it at the same time as you declare it.